language/de.json

```diff
@@ -0,0 +1,372 @@
+{
+	"Preview": "Vorschau",
+	"Gallery": "Galerie",
+	"Generate": "Generieren",
+	"Skip": "Überspringen",
+	"Stop": "Stop",
+	"Input Image": "Eingabebild",
+	"Advanced": "Erweitert",
+	"Upscale or Variation": "Vergrößern oder Variieren",
+	"Image Prompt": "Bild Prompt",
+	"Inpaint or Outpaint (beta)": "Inpainting oder Outpainting (beta)",
+	"Drag above image to here": "Ziehe das obere Bild hierher",
+	"Upscale or Variation:": "Vergrößern oder Variieren:",
+	"Disabled": "Deaktiviert",
+	"Vary (Subtle)": "Variieren (Subtil)",
+	"Vary (Strong)": "Variieren (Stark)",
+	"Upscale (1.5x)": "Vergrößern (1.5x)",
+	"Upscale (2x)": "Vergrößern (2x)",
+	"Upscale (Fast 2x)": "Vergrößern (Schnell 2x)",
+	"\ud83d\udcd4 Document": "\uD83D\uDCD4 Dokument",
+	"Image": "Bild",
+	"Stop At": "Anhalten bei",
+	"Weight": "Gewichtung",
+	"Type": "Typ",
+	"PyraCanny": "PyraCanny",
+	"CPDS": "CPDS",
+	"* \"Image Prompt\" is powered by Fooocus Image Mixture Engine (v1.0.1).": "* \"Bild Prompt\" wird von Fooocus Image Mixture Engine (v1.0.1) angetrieben.",
+	"The scaler multiplied to positive ADM (use 1.0 to disable).": "Skalierung multipliziert mit positivem ADM (1.0 verwenden, um zu deaktivieren).",
+	"The scaler multiplied to negative ADM (use 1.0 to disable).": "Skalierung multipliziert mit negativem ADM (1.0 verwenden, um zu deaktivieren).",
+	"When to end the guidance from positive/negative ADM.": "Wann der Leitfaden von positivem/negativem ADM beendet werden soll.",
```

Comment:
```suggestion
	"When to end the guidance from positive/negative ADM.": "Wann die Führung vom positivem/negativem ADM beendet werden soll.",
```

Reply:
As @Hangover3832 mentioned, guidance is rather difficult to translate gracefully so that it makes sense. I can implement your suggestion, but I would also want to hear @Hangover3832 thoughts about this

Reply:
Lenkung is also possible, I think Führung is rather the best choice.

Reply:
Alright, thank you. I will adjust it as necessary